### PR TITLE
Adding a release note for Alpha.

### PR DESCRIPTION
**Pull request by dinusha92**
### Purpose

Release note for alpha release.

### Issues

Fixes #

### Automation tests
 - Unit tests added: Yes/No
 - Integration tests added: Yes/No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: release-note.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                    and also offers a wide range of features which helps organizations to deploy APIs microservice
+                    architectures<br>
+                    efficiently.</p>
+            </li>
+        </ul>
+        <h2>Features</h2>
+        <ul>
+            <li>Choreo Connect provides <a
+                    href="https://github.com/wso2/product-microgateway/issues?q=is%3Aissue+project%3Awso2%2Fproduct-microgateway%2F8+is%3Aclosed+label%3A%22Type%2FNew+Feature%22">features</a>
+                for API Management.
```
Comment (Rajith90) [inline release-note.html:60]: Is this part required

Comment (dinusha92) [reply release-note.html:60]: We can just named it as features and point to the link if that's okay

Comment (dinusha92) [reply release-note.html]: Modified. Please have a look.

File: release-note.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                    applies the common quality of service attributes on API requests such as security, rate limiting and
+                    analytics<br>
+                    and also offers a wide range of features which helps organizations to deploy APIs microservice
+                    architectures<br>
+                    efficiently.</p>
+            </li>
+        </ul>
+        <h2>Features</h2>
+        <ul>
+            <li> Choreo connect <a
```
Comment (Rajith90) [inline release-note.html:58]:
```suggestion
            <li> Choreo Connect <a
```

**Review by Rajith90 — APPROVED** [review]

**Pull request merged**